File: tarot-website/src/getCard.js
window.addEventListener('load', () =>{
    //make request to get API data
    const req = new XMLHttpRequest();
    req.open('GET', 'https://tarotapi.almirpaulo.repl.co/'
, false)
    req.send()
    const res = JSON.parse(req.response)
    console.log(res)

    //get a random number to get a random card
   const pick = parseInt(Math.random() * 21)
    
//DOM elements variables
    const cardImg = document.getElementById('card-img');
    const cardTitle = document.getElementById('title');
    const questions = document.getElementById('questions');

    //Set new values
    cardTitle.innerText = res.cards[pick].title

    const card = document.createElement('IMG');
    const cardSrc = card.src
    cardSrc = res.cards[pick].picture
    const cardAlt = card.getAttribute('alt');
    cardAlt = 'card'
    card.id = 'card'


    

});

File: tarot-website/src/get_card.py
from browser import document as doc
from browser import ajax, html, console
from javascript import JSON 
import random


#variables
card_img = doc['card']
title = doc['title']
questions = doc['questions']

#Ajax call
def get_data(req):
    if req.status == 200 or req.status == 0:
        res = JSON.parse(req.response)
        pick = random.randint(0, 21)
        console.log(pick, res)
        title.innerText = res['cards'][pick]['title']
        card_img.attrs['src'] = res['cards'][pick]['picture']
        for q in res['cards'][pick]['questions']:
           questions <= html.LI(q)  
    else:
        print('error code 500')

req = ajax.Ajax()
req.bind('complete', get_data)
req.open('GET', 'https://tarotapi.almirpaulo.repl.co/', False)
req.send()


